components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -303,4 +303,16 @@ typedef void (^MDCActionSheetHandler)(MDCActionSheetAction *_Nonnull action);
  */
 @property(nonatomic, copy, nullable) UIColor *tintColor;
 
+/**
+ The color of the divider at the top of the action.
+ */
+@property(nonatomic, copy, nullable) UIColor *dividerColor;
+
+/**
+ A Boolean value that controls whether a divider is shown at the top of the action.
```

Comment:
Tip: You could remove "A boolean value that" to shorten the description without losing information.

Reply:
Done